matson.libraries.yml

```diff
@@ -103,6 +104,8 @@ matson:
   js:
     js/vendor/modernizr.js: {}
     js/build/scripts.js: {}
+    js/build/sidebar.js: {}
+    node_modules/scrolltofixed/jquery-scrolltofixed-min.js: {}
```

Comment:
On second thought - are you sure?  vendor directories are all in .gitignore.  How about /libraries instead?  That seems more drupal-y.